nix/workbench/run.sh

```diff
@@ -289,7 +298,12 @@ EOF
         )
         genesis prepare "${genesis_args[@]}"
 
-        backend start-run "$dir" "$@"
+        ## Record genesis.
+        cp "$dir"/genesis/genesis.json "$dir"/genesis.json
```

Comment:
Why do you need to do that ?
Why is this `cp` at this point in the script ?
IMHO having the same config twice in two files is confusing.

Reply:
It's necessary due to how genesis caching works.